### DISCUSSION TITLE
Why isn't my country on the list in the 2FA section (SMS/Text message)?

### DISCUSSION THREAD

**Discussion by pr0mming** [community/community#130241] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey guys, I have this screenshot:

![image](https://github.com/community/community/assets/20020612/c2164960-62cc-4b5b-ab91-d52ef76fca93)

And basically Colombia isn't on the list to continue adding a second 2FA, I don't know if there is another country missing.

**Comment by ebndev** [comment, marked as answer]
Hey there @pr0mming,

Thanks so much for reaching out in GitHub's Community! If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
